lib/cove/command/docker/container/create.rb

```diff
@@ -17,6 +17,10 @@ def self.build(image:, index: nil, name: nil, remove: false, interactive: false,
               end
             end
 
+            Array(volumes).each do |volume|
+              builder += ["--mount", "type=volume,", "source=" + volume["source"] + ",", "target=" + volume["target"]]
```

Comment:
The argument to `--mount` has to be a single string.

```suggestion
              builder += ["--mount", "type=volume,source=" + volume["source"] + "," + "target=" + volume["target"]]
```


Or even
```suggestion
              builder += ["--mount", "type=volume,source=#{volume["source"]},target=#{volume["target"]}"]
```